#MaplabIntegrator
maplabIntegratorVerbosity: 1                      #Verbosity level of MaplabIntegrator (0:quiet)
world_frame: darpa                                #World(Absolute) frame
map_frame: world                                  #Robot Map frame
base_frame: imu                                  #Base(IMU) frame
lidarFrame: imu                                  #LiDAR frame name - used for LiDAR-to-Sensor transform lookup
maplab_camera_frame: camera0                      #Frame of camera used for apriltag detection (absolute poses)
maplab_imu_frame: imu                            #Frame of IMU used by maplab 
publishWorldtoMapTransform: true                  #Publish World-to-Map transform. Used for debugging as Absolute constraint publisher will publish this trasnform
cloudSavePosDelta: 1.0                            #Delta translation (meters) occured before saving cloud
cloudSaveRotDelta: 0.2                            #Delta rotation (radian) occured before saving cloud
odometryFactorNoise: [0.05, 0.05, 0.05, 0.1, 0.1, 0.1]                       #Noise of Odometry factors - ORDER RPY(rad) - XYZ(m)
absoluteFactorNoise: [1.0e-06, 1.0e-06, 1.0e-06, 1.0e-06, 1.0e-06, 1.0e-06]  #Noise of Absolute factors at position 0- ORDER RPY(rad) - XYZ(m)
submapFactorNoise: [1.0e-03, 1.0e-03, 1.0e-03, 1.0e-03, 1.0e-03, 1.0e-03]    #Noise of relative Submap factors - ORDER RPY(rad) - XYZ(m)
anchorFactorNoise: [1.0e-05, 1.0e-05, 1.0e-05, 1.0e-05, 1.0e-05, 1.0e-05]    #Noise of anchor factors - ORDER RPY(rad) - XYZ(m) 
updateResultsInterval: 5.0                                                   #Interval(seconds) at which graph is optimized